app/controllers/council_memberships_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CouncilMembershipsController < ApplicationController
+
+	def index
+		@council_memberships = CouncilMembership.where(user_id: current_user.id)
+	end
+
+	def new
+		@council_membership = CouncilMembership.new
+	end
+
+	def create
+		@council_membership = CouncilMembership.where(council_membership_params).first_or_create
+    if @council_membership.save
+      send_notification("#{current_user.firstname}'s you to member #{@council_membership.council.council_name}", 'Council' )
+      redirect_to councils_path
+    else
+      flash[:error] = 'Field cannot be left blank.'
```

Comment:
This error report is presuming a particular error happened, but with no evidence - you should use the errors objects on the model to display what did go wrong, or just show the user a generic "something went wrong" message. You shouldn't presume you know the error if you don't. What if the problem was a dead database connection?